target/linux/ipq40xx/image/generic.mk

```diff
@@ -538,6 +538,18 @@ define Device/extreme-networks_ws-ap3915i
 endef
 TARGET_DEVICES += extreme-networks_ws-ap3915i
 
+define Device/extreme-networks_ws-ap391x
+	$(call Device/FitImage)
+	DEVICE_VENDOR := Extreme Networks
+	DEVICE_MODEL := WS-AP391x
+	IMAGE_SIZE := 30080k
+	SOC := qcom-ipq4029
+	BLOCKSIZE := 128k
+	IMAGE/sysupgrade.bin := append-kernel | append-rootfs | pad-rootfs | check-size | append-metadata
+	DEVICE_PACKAGES := tcpdump iperf3 luci openssh-client openssh-client-utils openssh-keygen
```

Comment:
Why are these packages required to be included by default?

Reply:
This AP has room for it, and it provides me with a better user experience with them.  I can remove though.

Reply:
Then please remove them, they are not required for the device to function and they can be easily installed later or selected during building.

Reply:
I've removed them